src/Config/BaseConfig.php

```diff
@@ -247,6 +267,19 @@ private function checkAndSetOverrides($config)
             $this->endpoint = (string)$config['endpoint'];
         }
 
+        $this->callbackUrl = '';
+
+        if (isset($config['callback_url']) && !empty($config['callback_url'])) {
+            $this->callbackUrl = (string)$config['callback_url'];
```

Comment:
Codecov is saying that this is the line that is causing the coverage drop.

<img width="790" alt="screen shot 2017-08-23 at 10 51 47 am" src="https://user-images.githubusercontent.com/3657251/29607322-29136f42-87f1-11e7-94ad-ce11dbc6e630.png">